crates/aptos/src/move_tool/mod.rs

```diff
@@ -467,11 +481,22 @@ impl CliCommand<&'static str> for TestPackage {
                 AbstractValueSizeGasParameters::zeros(),
             ),
             None,
-            false,
+            self.compute_coverage,
             &mut std::io::stdout(),
         )
         .map_err(|err| CliError::UnexpectedError(err.to_string()))?;
 
+        // Print coverage summary if --coverage is set
+        if self.compute_coverage {
+            config.test_mode = false;
+            let summary = CoverageSummaryOptions::Summary {
+                functions: false,
+                output_csv: false,
+            };
+            generate_coverage_info(summary, path.as_path(), config, self.filter).unwrap();
+            println!("Please use `aptos move coverage -h` for more detailed test coverage of this package");
```

Comment:
Anything you print out in the CLI, please make `eprintln` so one can just parse the output API JSON.